### PR TITLE
Changed the example for extend and added more plots

### PR DESCRIPTION
**Pull request by iamprayush**
#### References to other Issues or PRs

#15327

#### Brief description of what is fixed or changed
I changed the extend output example as it was similar to append. I also added the outputs for more plots. 

#### Release Notes

* plotting
  * changed extend example and added new plots

### CODE REVIEW

**Comment by iamprayush** [comment]
This is the output that I get:
[plots.pdf](https://github.com/sympy/sympy/files/2454260/plots.pdf)
If this looks proper, I can do the same for the rest of the plots.

**Comment by iamprayush** [comment]
@moorepants Could you help me understand why Travis CI is giving me an error on my last commit. Everything seems to work fine locally and the outputs are as expected.

**Comment by moorepants** [comment]
> This is the output that I get:

Please post screenshots of the HTML version.

**Comment by moorepants** [comment]
> Could you help me understand why Travis CI is giving me an error on my last commit.

The test didn't run due to a network error. I restarted it.

File: sympy/plotting/plot.py [hunk 1]
```diff
@@ -262,16 +262,22 @@ def extend(self, arg):
...
+        .. plot::
+           :format: doctest
+           :include-source: True
+
+           >>> from sympy import symbols
+           >>> from sympy.plotting import plot
+           >>> x = symbols('x')
+           >>> p1 = plot(x**2, show=False)
+           >>> p2 = plot(x, -x, show=False)
+           >>> p1.extend(p2)
```
Comment (moorepants) [inline sympy/plotting/plot.py]: I thought the difference in append and extend is that the one takes a singleton and the other takes a sequence, e.g. `.append('x')` and '.extend(['x', 'y'])`.

**Comment by iamprayush** [comment]
Yes, so in the .append() example, only the first index of p2 (which incidentally has only one plot) is appended. If there were many plots in p2, then we'd have to append each one separately by specifying the index of each plot. While in .extend() we can directly pass in p2 and we'd see all the plots of p2 get "joined/added" to p1. This is what I've understood, and I've made the example accordingly. I can add this kind of an explanation in the docs if that's helpful. 

And here are the screenshots of the output:

screenshot from 2018-10-09 12-31-23
screenshot from 2018-10-09 12-31-32
screenshot from 2018-10-09 12-31-46

**Pull request merged**